markdown/beginners_guide.md

```diff
@@ -11,30 +12,45 @@ It has been developed specifically to ease the creation and execution of bioinfo
 The benefits of having your pipeline in Nextflow include:
 
 * Built-in GitHub support.
-* Compatibility with virtually all computational infrastructures, including all major HPC job schedulers.
+* Compatibility with virtually all computational infrastructures, including all major cluster job schedulers.
 * Integrated software dependency management (Docker, Singularity, Conda).
 * Portability so you can run your pipeline anywhere laptop, cluster or cloud.
 * Reproducibility of analyses independent of time and computing platform.
 
 Whether your pipeline is a simple BLAST execution or a complex genome annotation pipeline, you can build it with Nextflow.
 
-# What is Docker?
-[Docker](https://www.docker.com/) is a tool to package and run applications locally in your machine within isolated environments called containers.
-Importantly, a Docker container images can be shared and, as such, all dependencies and versions required to run a specific script can be kept constant, even when a script is executed on a different machine, simply by running it in the respective Docker container.
+# Software requirements
+An analysis pipeline chains the execution of multiple tools together.
+In order for this to work, each of those software tools must be installed.
+Historically, this can be a source of great frustration and a key step where reproducibility between analyses is lost.
+All nf-core pipelines fully utilise the built-in support for software packaging that Nextflow offers.
+
+## Docker
+[Docker](https://www.docker.com/) is a tool to package and run software within isolated environments called containers.
+Importantly, a Docker container images can be shared. As such, all dependencies and versions required to run a specific script can be kept constant, even when a script is executed on a different machine, simply by running it in the respective Docker container.
 The usage of [Docker in Nextflow](https://www.nextflow.io/docs/latest/docker.html) and container technology enables us to:
 
 * Simplify the setup of complicated software, libraries and modules.
 * Make our results, analysis and graphs 100% reproducible.
 * Share our work with the world.
 
-## When not to use Docker?
-Docker is good when you are running your pipe locally. When you are working in a server it is recommended to work using [**Singularity**](https://www.sylabs.io/guides/2.5.1/user-guide/) containers.
```

Comment:
This isn't quite true. If people are using a HPC server with docker installed, then that's great! No need to use Singularity. It's just that most shared HPCs can't run docker.

A classic example is AWS, where basically everyone does (and should) use Docker.